iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/ThreeWindingsTransformerAdapter.java

```diff
@@ -95,6 +100,36 @@ public CurrentLimitsAdder newCurrentLimits() {
             return getDelegate().newCurrentLimits();
         }
 
+        @Override
+        public ApparentPowerLimits getApparentPowerLimits() {
+            return getDelegate().getApparentPowerLimits();
+        }
+
+        @Override
+        public ApparentPowerLimitsAdder newApparentPowerLimits() {
+            return getDelegate().newApparentPowerLimits();
+        }
+
+        @Override
+        public ActivePowerLimits getActivePowerLimits() {
+            return getDelegate().getActivePowerLimits();
+        }
+
+        @Override
+        public ActivePowerLimitsAdder newActivePowerLimits() {
+            return getDelegate().newActivePowerLimits();
+        }
+
+        @Override
+        public VoltageLimits getVoltageLimits() {
```

Comment:
I am not sure to understand how voltage limits work indeed. You associate it to the terminal connected at network side ? I imagine that it is similar for the other operational limits.